docs/src/api.md

```diff
@@ -0,0 +1,209 @@
+# API
+
+
+## Finding attractors
+
+Attractors.jl defines a generic interface for finding attractors of dynamical systems. One first decides the instance of [`DynamicalSystem`](@ref) they need. Then, an instance of [`AttractorMapper`](@ref) is created from this dynamical system. This `mapper` instance can be used to compute e.g., [`basins_of_attraction`](@ref), and the output can be further analyzed to get e.g., the [`basin_entropy`](@ref).
+
+```@docs
+AttractorMapper
+```
+
+
+### Recurrences
+
+```@docs
+AttractorsViaRecurrences
+automatic_Δt_basins
+SubdivisionBasedGrid
+subdivision_based_grid
+```
+
+### Proximity
+```@docs
+AttractorsViaProximity
+```
+
+### Featurizing
+```@docs
+AttractorsViaFeaturizing
+```
+
+
+## Grouping configurations
+
+Grouping configurations that can be given to [`AttractorsViaFeaturizing`](@ref)
+are part of a generic and extendable interface based on the [`group_features`](@ref)
+function. It can also be used and extended completely independently of finding
```

Comment:
```suggestion
function. The grouping configuration sets how the features describing the trajectories will be grouped together.  It can also be used and extended completely independently of finding
```